src/shorts/shorts.controller.js

```diff
@@ -57,3 +58,20 @@ export const createShorts = async (req, res, next) => {
         res.send(response(status.CREATED));
     });
 }
+
+export const addComment = async (req, res, next) => {
+    const { shorts_id } = req.query;
```

Comment:
명세 상에서 쇼츠 아이디 값을 쿼리 값으로 받아오는 것이 아니라 아래 예제처럼 파람 값으로 받아오는 거라서 이 부분 수정 부탁드려요!
```
{{base}}/shorts/3/comments
```

```suggestion
    const { shorts_id } = req.params.shortsId;
```